### DISCUSSION TITLE
Is it possible to disable animations for elements?

### DISCUSSION THREAD

**Discussion by summersz** [material-components/material-web#5139] [Q&A]
In Angular you can use [@.disabled] to prevent the default component animations from running.

Is this already possible? or if not, can it be added?

**Comment by asyncliz** [comment]
There isn't a unified way to do so. Some components have hooks (like dialog), while others set animations on their `:host` so they can be overridden with CSS (like elevation). Others are nested in shadow dom and not customizable.

We have goals for the next two quarters to work on a motion system for better control of animations :)
